### DISCUSSION TITLE
What playback speeds are actually useful?

### DISCUSSION THREAD

**Discussion by juriroemer** [advplyr/audiobookshelf-app#57] [General]
(The speed increments in the mobile app and the web app are different, so I'm not commenting about the actual speeds in use right now.)

From what I have noticed there is no real use in any playback speeds that are >`2x` (which only exist in the mobile app right now), because when applied to normal spoken word, there's so much is skipped that you can't comprehend the content anymore. On the flip side I would wish for a speed that is faster than 1x, but slower than 1.3x (which is the next step on mobile right now).

What I would propose is speeds of maybe

```
0.5x

0.8x

1x

1.1x

1.2x

1.3x

1.5x

1.7x
```

I know some platforms have speeds >=`2x`, but these service mostly also have video, where it can actually be useful to have these high speeds even if audible comprehension is lost.

I might be totally wrong on this and miss actual usecases for faster speeds, so I'm very interested on hearing some opinions on this.

**Comment by Jdiesel87** [comment]
I think the optimal speed depends largely on the narrator, content and how immersed you want to be in the content. I can't go very fast before I start to lose focus.

Another important thing is pitch/tempo correction. When you speed things up too much you start to get weird effects. Good correction is a necessity.

**Comment by advplyr** [comment]
One of the first requests actually was for playback speeds up to 3x.
I think the best solution is to make the mobile app like the web app where you can +/- the playback speed in increments 0.1

You would be able to select a speed between 0.5 and 3, which should meet even the most extreme speed listeners expectations.

**Reply by juriroemer** [reply]
My impression was actually that whatever algorithm is used to speed up the playback chops so much of the actual content instead of just speeding it up, that it becomes incomprehensible. But I might be wrong.

Having individual 0.1x steps sounds great!

**Comment by advplyr** [comment]
Playback speed controls are updated in `v0.9.34-beta`

![image](https://user-images.githubusercontent.com/67830747/146088866-32a0aeb8-0367-4bd3-b2e6-249f992e961f.png)

**Reply by argash** [reply]
These work for me, though I might change 1.2 to be 1.25? but yeah I spend most of my time at 1.5x

**Reply by tvories** [reply]
I would also agree to change to 1.25 though I doubt I could tell you the difference if I heard it :D

**Comment by hybridtracer** [comment]
My default is 2x and may go faster for certain narrators so certainly go to least 3x in my opinion.

**Reply by advplyr** [reply]
You should already be able to go to 3x playback speed on every client
